### ISSUE TITLE
How can I reproduce the results in the paper

### ISSUE THREAD

**Issue by Node2vec** [yuh-yang/KGCL-SIGIR22#2]
Thanks for your great work！I tried running the code but couldn't get the results in the paper,
especially the MIND dataset, which stopped early at the 7th epoch，the best result I got was
{'precision': array([0.00819516]), 'recall': array([0.07631685]), 'ndcg': array([0.03943752])}
It is far from the results in the paper.
Could you please provide some advice and training logs?
Thanks!

**Comment by thus10** [comment]
I have the same question.

**Comment by yuh-yang** [comment]
Hi, @thus10, @Node2vec. We've released the log of KGCL running on MIND without a validation set. Hope you guys find it helpful :).

**Comment by Node2vec** [comment]
Thank you!
I notice an inconsistency in the number of entities.
your log:user:300000, item:48957, entity:57188
the repo:user:300000, item:48957, entity:57473
Is it possible that the data set you use is different from the dataset in the repo？

**Comment by yuh-yang** [comment]
Hi guys, we've noticed that the released code has performance problem running with MIND. But it seems not caused by minor inconsistencies in the KG dataset (actually the kg.txt of MIND is generated with random sampling factors). We are closely working on it. Thanks for issuing.

**Comment by Node2vec** [comment]
Thank you very much! 
Can you provide the same three datasets as in the paper?
It seems that there are also some inconsistencies between the repo and the paper.

**Comment by yuh-yang** [comment]
Hi guys, we've fixed the performance bug with commit 10f1049. Please try that :).

**Comment by Node2vec** [comment]
Thanks!
